code/src/components/Footer.vue

```diff
@@ -1,13 +1,163 @@
 <template>
-  <h2>asdasda</h2>
+  <footer>
+    <div class="footer-mainSection">
+      <div class="footer-leftSection">
+        <p class="footer-title">@FLUXACADEMY</p>
+        <p>
+          @FLUXACADEMY Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent a
+          viverra magna. Maecenas vitae est nunc. Phasellus vitae odio posuere, cursus
+          purus id, faucibus metus. Cras posuere porttitor elit. Etiam dignissim, purus
+          nec suscipit pretium.
+        </p>
+      </div>
+      <div class="footer-middleSection">
+        <p class="footer-title">CONTÁCTANOS</p>
+        <p>
+          TELÉFONO: +57 312-578-4791 <br />
+          EMAIL: flux@prueba.com <br />WHATSAPP:
+          <a
+            class="footer-whatsAppLink"
+            href="https://api.whatsapp.com/send?phone=+57 312 578 4791&text=Hola%20me%20gustaría%20saber%20más"
+          >
+            Click aquí
+          </a>
+        </p>
+      </div>
+      <div class="footer-rightSection">
+        <p class="footer-title">Siguenos</p>
+        <div class="footer-icons">
+          <a href="https://www.facebook.com/pg/Flux-academy-103864354337166/posts/"
+            ><i class="fab fa-facebook fa-lg"></i
+          ></a>
+          <a href="https://www.instagram.com/flux.parkour/">
+            <i class="fab fa-instagram fa-lg"></i>
+          </a>
+          <a href="https://www.youtube.com/user/PlusParkour">
+            <i class="fab fa-youtube fa-lg"></i>
+          </a>
+          <a href="https://github.com/monotera/FluxWebPage">
+            <i class="fab fa-github fa-lg"></i>
+          </a>
+        </div>
+      </div>
+    </div>
+    <p class="footer-copyRight">
+      2020 Flux Academy by
+      <a class="footer-copyRightLink" href="https://github.com/Manuel-2011"
+        >Manuel Mosquera</a
+      >
+      y
+      <a class="footer-copyRightLink" href="https://github.com/monotera"
+        >Nelson Mosquera</a
+      >
+    </p>
+  </footer>
 </template>
-
 <script>
 export default {};
 </script>
+<style lang="scss" scoped>
+* {
+  margin-bottom: 0.5rem;
+  text-align: center;
+  @media screen and (min-width: $breakpoint-desktop) {
+    text-align: start;
+  }
+}
+
+footer {
+  color: white;
+  background: black;
+  display: inline-block;
+  font-family: "fontMontserratRegular", sans-serif;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+
+  .footer-mainSection {
+    @media screen and (min-width: $breakpoint-desktop) {
+      display: flex;
+      width: 100%;
+      justify-content: space-evenly;
+      align-items: flex-start;
+    }
+  }
+
+  .footer-title {
+    font-family: "fontMontserratBold", sans-serif;
```

Comment:
I think we should use an scss variable to determine the styles we are going to use in our web page, so `headers-font` would be a good fit for this one. This will allow us to change all the webpage font styles with a few changes in the scss variables.

Reply:
I have made the required changes